packages/mutation-testing-elements/package-lock.json

```diff
@@ -1,7 +1,30 @@
 {
-  "name": "mutation-testing-elements",
-  "version": "1.2.3",
-  "lockfileVersion": 1,
-  "requires": true,
-  "dependencies": {}
+	"name": "mutation-testing-elements",
+	"version": "1.2.3",
+	"lockfileVersion": 1,
+	"requires": true,
+	"dependencies": {
+		"@stryker-mutator/api": {
+			"version": "2.5.0",
+			"resolved": "https://registry.npmjs.org/@stryker-mutator/api/-/api-2.5.0.tgz",
+			"integrity": "sha512-6z2AZeGcI+ZzqZEBX6h+vWt11uBzLfAg24kV0b/CJZ69zNELRLkjJnLaE4aU1o5uE6OyFGKJJ0OVxtZrnvZSuQ==",
+			"dev": true,
+			"requires": {
+				"mutation-testing-report-schema": "^1.1.0",
+				"tslib": "~1.10.0"
+			}
+		},
+		"mutation-testing-report-schema": {
```

Comment:
Is this intended?

Reply:
I've added it in JS Doc here: https://github.com/stryker-mutator/mutation-testing-elements/blob/chore/eslint/packages/mutation-testing-elements/tsconfig-transpiler.js#L13

But we could remove it as well. Not really needed.

Reply:
I meant the mutation-testing-report-schema dependency. I thought Lerna was supposed to take care of that instead of the published package being in the package-lock